README.md

```diff
@@ -21,6 +21,19 @@
 
 `iceberg` is a Golang implementation of the [Iceberg table spec](https://iceberg.apache.org/spec/).
 
+## Build From Source
+
+### Prerequisites
+
+* Go 1.21 or later
+
+### Build
+
+```shell
+$ git clone https://github.com/apache/iceberg-go.git
+$ cd iceberg-go/cmd/iceberg && go build .
+```
```

Comment:
you could just do `go install github.com/apache/iceberg-go/cmd/iceberg` if you don't intend to work on it yourself. there's no direct need to manually clone the git repo

Reply:
@zeroshade Thanks for your reply. This section is for building from the source, so perhaps it would be better to clone and build instead of installing. What do you think?

Reply:
That makes sense, and is fine. Though I'm not sure it is necessarily needed to explain for any developers if they are going to work on it. But i guess it's fine.